site/content/en/docs/manual/advanced/dataset_manifest.md

```diff
@@ -11,11 +11,62 @@ description: This section on [GitHub](https://github.com/cvat-ai/cvat/tree/devel
 
 <!--lint disable heading-style-->
 
-### Steps before use
+## Overview
```

Comment:
I think the page title should also be adjusted, because it's not just about the command line anymore.